static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -0,0 +1,160 @@
+  const hydratedFlags = useMemo(
+    () =>
+      hydrateFlags(
+        event.contexts?.flags?.values.filter(
+          (a, i, arr) => arr.findIndex(b => b.flag === a.flag) === i
+        )
+      ),
+    [event]
+  );
+
+  const handleSortAlphabetical = (flags: KeyValueDataContentProps[]) => {
+    return [...flags].sort((a, b) => {
+      return a.item.key.localeCompare(b.item.key);
+    });
+  };
+
+  const sortedFlags =
+    sortMethod === FlagSort.ALPHA ? handleSortAlphabetical(hydratedFlags) : hydratedFlags;
+
+  const onViewAllFlags = useCallback(() => {
+    openDrawer(
+      () => (
+        <FeatureFlagDrawer
+          group={group}
+          event={event}
+          project={project}
+          hydratedFlags={hydratedFlags}
+          initialSort={sortMethod}
+        />
+      ),
+      {
+        ariaLabel: t('Feature flags drawer'),
+        // We prevent a click on the 'View All' button from closing the drawer so that
+        // we don't reopen it immediately, and instead let the button handle this itself.
+        shouldCloseOnInteractOutside: element => {
+          const viewAllButton = viewAllButtonRef.current;
+          if (viewAllButton?.contains(element)) {
+            return false;
+          }
+          return true;
+        },
+        transitionProps: {stiffness: 1000},
+      }
+    );
+  }, [openDrawer, event, group, project, sortMethod, hydratedFlags]);
+
+  if (!hydratedFlags.length) {
+    return null;
+  }
+
+  const actions = (
+    <ButtonBar gap={1}>
+      <Button
+        size="xs"
+        aria-label={t('View All')}
+        ref={viewAllButtonRef}
+        onClick={() => {
+          isDrawerOpen ? closeDrawer() : onViewAllFlags();
+        }}
+      >
+        {t('View All')}
+      </Button>
+      <CompactSelect
+        value={sortMethod}
+        options={FLAG_SORT_OPTIONS}
+        triggerProps={{
+          'aria-label': t('Sort Flags'),
+        }}
+        onChange={selection => {
+          setSortMethod(selection.value);
+        }}
+        trigger={triggerProps => (
+          <DropdownButton {...triggerProps} size="xs" icon={<IconSort />}>
+            {getLabel(sortMethod)}
+          </DropdownButton>
+        )}
+      />
+      <Button
+        size="xs"
+        icon={<IconChevron direction={isCollapsed ? 'down' : 'up'} />}
+        aria-label={t('Collapse Section')}
+        onClick={() => {
+          setIsCollapsed(!isCollapsed);
+        }}
+        borderless
+      />
+    </ButtonBar>
+  );
+
+  // Split the flags list into two columns for display
+  const truncatedItems = sortedFlags.slice(0, 20);
+  const columnOne = truncatedItems.slice(0, 10);
+  let columnTwo: typeof truncatedItems = [];
+  if (truncatedItems.length > 10) {
+    columnTwo = truncatedItems.slice(10, 20);
+  }
+
+  return (
+    <ErrorBoundary mini message={t('There was a problem loading feature flags.')}>
+      <EventDataSection title={t('Feature Flags')} type="feature-flags" actions={actions}>
```

Comment:
can you use the InterimSection component here for the new layout

![image](https://github.com/user-attachments/assets/7010429b-8b4f-41ef-9a20-065cd417a744)


```suggestion
      <InterimSection title={t('Feature Flags')} type="feature-flags" actions={actions}>
```

Reply:
yup just switched!